### ISSUE TITLE
Time in error bar should match time  of the last error reported

### ISSUE THREAD

**Issue by jwjsystem** [sdobreff/advanced-analytics#8]
Right now, this time here will get updated once the page refreshes based on the system clock. 

I suggest (or its a bug) that this timer should remain the time when the error last occured, so that we can easily correlate that time , with an error. 

Otherwise, somebody can be confused that there was as new error after refreshing the page (since time also updates) etc. 

![xxx](https://github.com/user-attachments/assets/86c36b58-b6bb-4731-9e5a-145e9c36a883)

**Comment by sdobreff** [comment]
You are correct - actually that is not the system time but the time of the last error detected, but localized using the WP settings - the time in table is the logged time on server using PHP timezone settings - anyway - good discovery
